src/sprites.asm

```diff
@@ -616,7 +616,7 @@ GradeRendering::
     ld [wGrade1+1], a
```

Comment:
$91 and $17 above this line are magic numbers. Please pull in the current master branch and change these to GRADE_BASE_X and GRADE_BASE_Y.